frontend/pages/hubs/[hubUrl].tsx

```diff
@@ -288,101 +294,103 @@ export default function Hub({
         customFooterImage={hubData.custom_footer_image && getImageUrl(hubData.custom_footer_image)}
         isLocationHub={isLocationHub}
       >
-        <div className={classes.content}>
-          {<DonationCampaignInformation />}
-          {!isLocationHub && (
-            <NavigationSubHeader
-              type={"hub"}
-              hubName={name}
-              allHubs={allHubs}
-              isLocationHub={isLocationHub}
-              hubUrl={hubUrl}
-              navigationRequested={navRequested}
-            />
-          )}
-          {!isLocationHub && (
-            <HubHeaderImage
-              image={getImageUrl(image)}
-              source={image_attribution}
-              onClose={closeHubHeaderImage}
-              isLocationHub={isLocationHub}
+        <ThemeProvider theme={hubThemeData ? transformThemeData(hubThemeData) : theme}>
```

Comment:
Used ThemeProvider to customize the hub page theme.

Reply:
Why don't we wrap `WideLayout` with `ThemeProvider` but only put it on this level?